src/nls/it/strings.js

```diff
@@ -648,7 +648,104 @@ define({
     "EXPAND_ALL"                    : "Espandi tutto",
     "COLLAPSE_CURRENT"              : "Comprimi attuale",
     "EXPAND_CURRENT"                : "Espandi attuale"
+    
+    // Descriptions of core preferences
+    "DESCRIPTION_CLOSE_BRACKETS"                     : "Vero per chiudere automaticamente parentesi graffe, staffe e parentesi",
+    "DESCRIPTION_CLOSE_OTHERS_ABOVE"                 : "Falso per rimuovere le Chiudere Altri Sopra dal menu contestuale di file di lavoro",
```

Comment:
`false per rimuovere il comando `**`Chiudi gli Altri`**` dal menù dello spazio di lavoro.`

I think `false` is better than `Falso` because `false` is the JSON value